app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable, which means it is shared across all instances of the `Person` class. If this shared behavior is not intended, consider using an instance variable instead to avoid potential issues.